### ISSUE TITLE
Can not detect issue when check dir

### ISSUE THREAD

**Issue by daixiang0** [golangci/golangci-lint#1537]
In https://github.com/daixiang0/kubeedge/tree/test-ci, `edge/pkg/servicebus/config/config.go` has an obvious issue, while do not detect it.

- [X] Yes, I'm using a binary release within 2 latest major releases. Only such installations are supported.
- [x] Yes, I've searched similar issues on GitHub and didn't find any.
- [X] Yes, I've included all information below (version, config, etc).

Please include the following information:

<details><summary>Version of golangci-lint</summary>

```console
$ golangci-lint --version
golangci-lint has version 1.30.0 built from 45b90f6 on 2020-08-03T03:10:34Z

```

</details>

<details><summary>Config file</summary>

https://github.com/daixiang0/kubeedge/blob/test-ci/.golangci.yml

</details>

<details><summary>Go environment</summary>

```console
$ go version && go env
go version go1.14.4 linux/amd64
GO111MODULE=""
GOARCH="amd64"
GOBIN=""
GOCACHE="/root/.cache/go-build"
GOENV="/root/.config/go/env"
GOEXE=""
GOFLAGS=""
GOHOSTARCH="amd64"
GOHOSTOS="linux"
GOINSECURE=""
GONOPROXY=""
GONOSUMDB=""
GOOS="linux"
GOPATH="/root/gopkg"
GOPRIVATE=""
GOPROXY="https://goproxy.cn,https://proxy.golang.org,https://goproxy.io,direct"
GOROOT="/root/go"
GOSUMDB="gosum.io+ce6e7565+AY5qEHUk/qmHc5btzW45JVoENfazw8LielDsaI+lEbq6"
GOTMPDIR=""
GOTOOLDIR="/root/go/pkg/tool/linux_amd64"
GCCGO="gccgo"
AR="ar"
CC="gcc"
CXX="g++"
CGO_ENABLED="1"
GOMOD="/root/gopkg/src/github.com/daixiang0/222/go.mod"
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -m64 -pthread -fmessage-length=0 -fdebug-prefix-map=/tmp/go-build843064397=/tmp/go-build -gno-record-gcc-switches"

```

</details>

<details><summary>Verbose output of running</summary>

```console
$ golangci-lint cache clean
$ golangci-lint run -v
INFO [config_reader] Used config file .golangci.yml
INFO [lintersdb] Active 9 linters: [errcheck gci goconst gofmt golint gosimple govet misspell whitespace]
INFO [loader] Go packages loading at mode 575 (exports_file|files|types_sizes|compiled_files|deps|imports|name) took 1.817608286s
INFO [runner/filename_unadjuster] Pre-built 0 adjustments in 45.374535ms
INFO [linters context/goanalysis] analyzers took 3m17.892640749s with top 10 stages: buildir: 2m59.131177356s, inspect: 4.870618839s, gci: 2.039499399s, printf: 1.894351344s, ctrlflow: 1.799971088s, gofmt: 987.955193ms, golint: 702.840023ms, misspell: 605.146607ms, whitespace: 365.885053ms, S1038: 255.550992ms
WARN [linters context] Panic: buildir: package "mappercommon" (isInitialPkg: true, needAnalyzeSource: true): runtime error: invalid memory address or nil pointer dereference: goroutine 21927 [running]:
runtime/debug.Stack(0x10fea98, 0x3c, 0xc008def0d8)
	/opt/hostedtoolcache/go/1.14.6/x64/src/runtime/debug/stack.go:24 +0x9d
github.com/golangci/golangci-lint/pkg/golinters/goanalysis.(*action).analyzeSafe.func1(0xc002189880)
	/home/runner/work/golangci-lint/golangci-lint/pkg/golinters/goanalysis/runner.go:508 +0x1b5
panic(0xf188a0, 0x1a0edb0)
	/opt/hostedtoolcache/go/1.14.6/x64/src/runtime/panic.go:969 +0x166
honnef.co/go/tools/ir.postDomFrontier.add(...)
	/home/runner/go/pkg/mod/honnef.co/go/tools@v0.0.1-2020.1.4/ir/lift.go:103
honnef.co/go/tools/ir.postDomFrontier.build(0xc001a97380, 0x8, 0x8, 0xc008dcd680)
	/home/runner/go/pkg/mod/honnef.co/go/tools@v0.0.1-2020.1.4/ir/lift.go:112 +0x8f
honnef.co/go/tools/ir.buildPostDomFrontier(...)
	/home/runner/go/pkg/mod/honnef.co/go/tools@v0.0.1-2020.1.4/ir/lift.go:122
honnef.co/go/tools/ir.lift(0xc008dcd680)
	/home/runner/go/pkg/mod/honnef.co/go/tools@v0.0.1-2020.1.4/ir/lift.go:208 +0x523
honnef.co/go/tools/ir.(*Function).finishBody(0xc008dcd680)
	/home/runner/go/pkg/mod/honnef.co/go/tools@v0.0.1-2020.1.4/ir/func.go:565 +0x36c
honnef.co/go/tools/ir.(*builder).buildFunction(0xc008defa68, 0xc008dcd680)
	/home/runner/go/pkg/mod/honnef.co/go/tools@v0.0.1-2020.1.4/ir/builder.go:2313 +0x477
honnef.co/go/tools/ir.(*builder).buildFuncDecl(0xc008defa68, 0xc00804ba70, 0xc00b052540)
	/home/runner/go/pkg/mod/honnef.co/go/tools@v0.0.1-2020.1.4/ir/builder.go:2334 +0xfa
honnef.co/go/tools/ir.(*Package).build(0xc00804ba70)
	/home/runner/go/pkg/mod/honnef.co/go/tools@v0.0.1-2020.1.4/ir/builder.go:2440 +0xcb8
sync.(*Once).doSlow(0xc00804bac8, 0xc00863dbd0)
	/opt/hostedtoolcache/go/1.14.6/x64/src/sync/once.go:66 +0xec
sync.(*Once).Do(...)
	/opt/hostedtoolcache/go/1.14.6/x64/src/sync/once.go:57
honnef.co/go/tools/ir.(*Package).Build(0xc00804ba70)
	/home/runner/go/pkg/mod/honnef.co/go/tools@v0.0.1-2020.1.4/ir/builder.go:2358 +0x65
honnef.co/go/tools/internal/passes/buildir.run(0xc016546f00, 0x43bbc0ae0, 0x1b1d5a0, 0xc0025e04c8, 0x2)
	/home/runner/go/pkg/mod/honnef.co/go/tools@v0.0.1-2020.1.4/internal/passes/buildir/buildir.go:89 +0x290
github.com/golangci/golangci-lint/pkg/golinters/goanalysis.(*action).analyze(0xc002189880)
	/home/runner/work/golangci-lint/golangci-lint/pkg/golinters/goanalysis/runner.go:590 +0xa25
github.com/golangci/golangci-lint/pkg/golinters/goanalysis.(*action).analyzeSafe.func2()
	/home/runner/work/golangci-lint/golangci-lint/pkg/golinters/goanalysis/runner.go:512 +0x2a
github.com/golangci/golangci-lint/pkg/timeutils.(*Stopwatch).TrackStage(0xc0018e73b0, 0x10519e5, 0x7, 0xc0042bd770)
	/home/runner/work/golangci-lint/golangci-lint/pkg/timeutils/stopwatch.go:111 +0x50
github.com/golangci/golangci-lint/pkg/golinters/goanalysis.(*action).analyzeSafe(0xc002189880)
	/home/runner/work/golangci-lint/golangci-lint/pkg/golinters/goanalysis/runner.go:511 +0x91
github.com/golangci/golangci-lint/pkg/golinters/goanalysis.(*loadingPackage).analyze.func2(0xc000f924b0, 0xc002189880)
	/home/runner/work/golangci-lint/golangci-lint/pkg/golinters/goanalysis/runner.go:1059 +0x61
created by github.com/golangci/golangci-lint/pkg/golinters/goanalysis.(*loadingPackage).analyze
	/home/runner/work/golangci-lint/golangci-lint/pkg/golinters/goanalysis/runner.go:1054 +0x306
INFO [runner] processing took 6.139µs with stages: max_same_issues: 1.002µs, nolint: 740ns, skip_dirs: 545ns, path_prefixer: 470ns, source_code: 455ns, diff: 439ns, cgo: 336ns, max_from_linter: 299ns, uniq_by_line: 253ns, filename_unadjuster: 251ns, skip_files: 242ns, autogenerated_exclude: 238ns, path_prettifier: 230ns, identifier_marker: 160ns, exclude: 93ns, exclude-rules: 81ns, max_per_file_from_linter: 80ns, path_shortener: 79ns, severity-rules: 76ns, sort_results: 70ns
INFO [runner] linters took 1m2.120662367s with stages: goanalysis_metalinter: 1m2.120603939s
INFO File cache stats: 439 entries of total size 2.2MiB
INFO Memory: 461 samples, avg is 955.9MB, max is 1223.8MB
INFO Execution took 1m3.991158686s
```

</details>

**Comment by daixiang0** [comment]
please help me @golangci/team

**Comment by ernado** [comment]
I see a panic in log, probably it is the primary issue?

**Comment by ldez** [comment]
It's an issue with [staticcheck](https://github.com/dominikh/go-tools)

```console
$ ./staticcheck ./...
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x70782f]

goroutine 117164 [running]:
honnef.co/go/tools/ir.postDomFrontier.add(...)
        honnef.co/go/tools@v0.0.1-2020.1.6/ir/lift.go:103
honnef.co/go/tools/ir.postDomFrontier.build(0xc006fcc6c0, 0x8, 0x8, 0xc021c44a20)
        honnef.co/go/tools@v0.0.1-2020.1.6/ir/lift.go:112 +0x8f
honnef.co/go/tools/ir.buildPostDomFrontier(...)
        honnef.co/go/tools@v0.0.1-2020.1.6/ir/lift.go:122
honnef.co/go/tools/ir.lift(0xc021c44a20)
        honnef.co/go/tools@v0.0.1-2020.1.6/ir/lift.go:208 +0x523
honnef.co/go/tools/ir.(*Function).finishBody(0xc021c44a20)
        honnef.co/go/tools@v0.0.1-2020.1.6/ir/func.go:565 +0x36c
honnef.co/go/tools/ir.(*builder).buildFunction(0xc0136cb5a0, 0xc021c44a20)
        honnef.co/go/tools@v0.0.1-2020.1.6/ir/builder.go:2313 +0x477
honnef.co/go/tools/ir.(*builder).buildFuncDecl(0xc0136cb5a0, 0xc020aed3b0, 0xc0133de540)
        honnef.co/go/tools@v0.0.1-2020.1.6/ir/builder.go:2334 +0xfa
honnef.co/go/tools/ir.(*Package).build(0xc020aed3b0)
        honnef.co/go/tools@v0.0.1-2020.1.6/ir/builder.go:2440 +0xcb8
sync.(*Once).doSlow(0xc020aed408, 0xc01d4d7708)
        sync/once.go:66 +0xec
sync.(*Once).Do(...)
        sync/once.go:57
honnef.co/go/tools/ir.(*Package).Build(0xc020aed3b0)
        honnef.co/go/tools@v0.0.1-2020.1.6/ir/builder.go:2358 +0x65
honnef.co/go/tools/internal/passes/buildir.run(0xc019c920c0, 0x244fff2ce, 0xf93f00, 0x41183d, 0xc0033ee918)
        honnef.co/go/tools@v0.0.1-2020.1.6/internal/passes/buildir/buildir.go:89 +0x290
honnef.co/go/tools/lint.(*Runner).runAnalysisUser(0xc00020bf40, 0xc019c920c0, 0xc0195b7b30, 0x8, 0xc0033eeb60, 0x0, 0xc0195b7b30)
        honnef.co/go/tools@v0.0.1-2020.1.6/lint/runner.go:494 +0x2af
honnef.co/go/tools/lint.(*Runner).runAnalysis(0xc00020bf40, 0xc0195b7b30, 0x0, 0x0, 0x0, 0x0)
        honnef.co/go/tools@v0.0.1-2020.1.6/lint/runner.go:374 +0x587
honnef.co/go/tools/lint.(*Runner).runAnalysisUser(0xc00020bf40, 0xc019c92000, 0xc018d7cb90, 0xc01e82c480, 0xc01e82c7e0, 0xc01e82c900, 0xc01e82d200)
        honnef.co/go/tools@v0.0.1-2020.1.6/lint/runner.go:480 +0xe8
honnef.co/go/tools/lint.(*Runner).runAnalysis(0xc00020bf40, 0xc018d7cb90, 0x0, 0x0, 0x0, 0x0)
        honnef.co/go/tools@v0.0.1-2020.1.6/lint/runner.go:374 +0x587
honnef.co/go/tools/lint.(*Runner).processPkg.func3(0xc006ac14b0, 0xc002e78480, 0xc000252750, 0xc00020bf40, 0xc018d7cb90, 0xc0195de000, 0x88, 0x88, 0x8)
        honnef.co/go/tools@v0.0.1-2020.1.6/lint/runner.go:913 +0x79
created by honnef.co/go/tools/lint.(*Runner).processPkg
        honnef.co/go/tools@v0.0.1-2020.1.6/lint/runner.go:908 +0x9f3
```

I recommend opening an issue in the [staticcheck](https://github.com/dominikh/go-tools) repository.

**Comment by daixiang0** [comment]
@ernado  @ldez is it normal that one failed linter does impact on other linters' result?

**Comment by ldez** [comment]
> is it normal that one failed linter does impact on other linters' result?

The panic comes from the goanalysis related to the MetaLinter, in this case, the global `recover` doesn't allow to continue the analysis.

- global recover:
https://github.com/golangci/golangci-lint/blob/cf32a7b7067315bc4c04a9ef5797ec71a8d9304e/pkg/lint/runner.go#L91-L101

- recover in goanalysis:
https://github.com/golangci/golangci-lint/blob/cf32a7b7067315bc4c04a9ef5797ec71a8d9304e/pkg/golinters/goanalysis/runner.go#L504-L514

- metalinter:
https://github.com/golangci/golangci-lint/blob/cf32a7b7067315bc4c04a9ef5797ec71a8d9304e/pkg/lint/lintersdb/enabled_set.go#L154

I see 2 solutions:
- throw the error
- create a "fake" `result.Issue`

**Comment by daixiang0** [comment]
I would pick the second one: create a result like "xxx linter panic, please file an issue to github" then continue run other linters.

**Comment by ldez** [comment]
The second solution create a dirty output:

<details>
<summary>logs</summary>

```
WARN [linters context] Panic: buildir: package "mappercommon" (isInitialPkg: true, needAnalyzeSource: true): runtime error: invalid memory address or nil pointer dereference: goroutine 22005 [running]:
runtime/debug.Stack(0xf35061, 0x3c, 0xc00d2d90c0)
        runtime/debug/stack.go:24 +0x9f
github.com/golangci/golangci-lint/pkg/golinters/goanalysis.(*action).analyzeSafe.func1(0xc006b680b0)
        github.com/golangci/golangci-lint/pkg/golinters/goanalysis/runner.go:508 +0x1be
panic(0xd81560, 0x1568e10)
        runtime/panic.go:969 +0x1b9
honnef.co/go/tools/ir.postDomFrontier.add(...)
        honnef.co/go/tools@v0.0.1-2020.1.6/ir/lift.go:103
honnef.co/go/tools/ir.postDomFrontier.build(0xc003563500, 0x8, 0x8, 0xc015ca8d80)
        honnef.co/go/tools@v0.0.1-2020.1.6/ir/lift.go:112 +0x8f
honnef.co/go/tools/ir.buildPostDomFrontier(...)
        honnef.co/go/tools@v0.0.1-2020.1.6/ir/lift.go:122
honnef.co/go/tools/ir.lift(0xc015ca8d80)
        honnef.co/go/tools@v0.0.1-2020.1.6/ir/lift.go:208 +0x525
honnef.co/go/tools/ir.(*Function).finishBody(0xc015ca8d80)
        honnef.co/go/tools@v0.0.1-2020.1.6/ir/func.go:565 +0x356
honnef.co/go/tools/ir.(*builder).buildFunction(0xc00d2d9a48, 0xc015ca8d80)
        honnef.co/go/tools@v0.0.1-2020.1.6/ir/builder.go:2313 +0x485
honnef.co/go/tools/ir.(*builder).buildFuncDecl(0xc00d2d9a48, 0xc015a963f0, 0xc015cd66f0)
        honnef.co/go/tools@v0.0.1-2020.1.6/ir/builder.go:2334 +0xff
honnef.co/go/tools/ir.(*Package).build(0xc015a963f0)
        honnef.co/go/tools@v0.0.1-2020.1.6/ir/builder.go:2440 +0xcc5
sync.(*Once).doSlow(0xc015a96448, 0xc01c86fbb0)
        sync/once.go:66 +0xec
sync.(*Once).Do(...)
        sync/once.go:57
honnef.co/go/tools/ir.(*Package).Build(0xc015a963f0)
        honnef.co/go/tools@v0.0.1-2020.1.6/ir/builder.go:2358 +0x65
honnef.co/go/tools/internal/passes/buildir.run(0xc013980f70, 0x299dd2ba1, 0x1676100, 0xc010156678, 0x2)
        honnef.co/go/tools@v0.0.1-2020.1.6/internal/passes/buildir/buildir.go:89 +0x298
github.com/golangci/golangci-lint/pkg/golinters/goanalysis.(*action).analyze(0xc006b680b0)
        github.com/golangci/golangci-lint/pkg/golinters/goanalysis/runner.go:590 +0xa83
github.com/golangci/golangci-lint/pkg/golinters/goanalysis.(*action).analyzeSafe.func2()
        github.com/golangci/golangci-lint/pkg/golinters/goanalysis/runner.go:512 +0x2a
github.com/golangci/golangci-lint/pkg/timeutils.(*Stopwatch).TrackStage(0xc001727310, 0xe8ccb4, 0x7, 0xc005408770)
        github.com/golangci/golangci-lint/pkg/timeutils/stopwatch.go:111 +0x50
github.com/golangci/golangci-lint/pkg/golinters/goanalysis.(*action).analyzeSafe(0xc006b680b0)
        github.com/golangci/golangci-lint/pkg/golinters/goanalysis/runner.go:511 +0x91
github.com/golangci/golangci-lint/pkg/golinters/goanalysis.(*loadingPackage).analyze.func2(0xc01e7b2600, 0xc006b680b0)
        github.com/golangci/golangci-lint/pkg/golinters/goanalysis/runner.go:1059 +0x65
created by github.com/golangci/golangci-lint/pkg/golinters/goanalysis.(*loadingPackage).analyze
        github.com/golangci/golangci-lint/pkg/golinters/goanalysis/runner.go:1054 +0x305 
WARN [runner] Can't process result by autogenerated_exclude processor: can't filter issue result.Issue{FromLinter:"goanalysis_metalinter", Text:"buildir: package \"mappercommon\" (isInitialPkg: true, needAnalyzeSource: true): runtime error: invalid memory address or nil pointer dereference", Severity:"error", SourceLines:[]string(nil), Replacement:(*result.Replacement)(nil), Pkg:(*packages.Package)(nil), LineRange:(*result.Range)(nil), Pos:token.Position{Filename:"goanalysis_metalinter", Offset:0, Line:0, Column:0}, HunkPos:0, ExpectNoLint:false, ExpectedNoLintLinter:""}: failed to get doc of file goanalysis_metalinter: failed to parse file: open goanalysis_metalinter: no such file or directory 
WARN [runner/source_code] Failed to get line 0 for file goanalysis_metalinter: failed to get file goanalysis_metalinter lines cache: can't get file goanalysis_metalinter bytes from cache: can't read file goanalysis_metalinter: open goanalysis_metalinter: no such file or directory 
goanalysis_metalinter:0: buildir: package "mappercommon" (isInitialPkg: true, needAnalyzeSource: true): runtime error: invalid memory address or nil pointer dereference (goanalysis_metalinter)
```

</details>

I would prefer the first solution:

<details>
<summary>logs</summary>

```
WARN [linters context] Panic: buildir: package "mappercommon" (isInitialPkg: true, needAnalyzeSource: true): runtime error: invalid memory address or nil pointer dereference: goroutine 21934 [running]:
runtime/debug.Stack(0xf35061, 0x3c, 0xc0021550c0)
        runtime/debug/stack.go:24 +0x9f
github.com/golangci/golangci-lint/pkg/golinters/goanalysis.(*action).analyzeSafe.func1(0xc0074e7130)
        github.com/golangci/golangci-lint/pkg/golinters/goanalysis/runner.go:508 +0x1be
panic(0xd81560, 0x1568e10)
        runtime/panic.go:969 +0x1b9
honnef.co/go/tools/ir.postDomFrontier.add(...)
        honnef.co/go/tools@v0.0.1-2020.1.6/ir/lift.go:103
honnef.co/go/tools/ir.postDomFrontier.build(0xc00d248a80, 0x8, 0x8, 0xc00b3e0000)
        honnef.co/go/tools@v0.0.1-2020.1.6/ir/lift.go:112 +0x8f
honnef.co/go/tools/ir.buildPostDomFrontier(...)
        honnef.co/go/tools@v0.0.1-2020.1.6/ir/lift.go:122
honnef.co/go/tools/ir.lift(0xc00b3e0000)
        honnef.co/go/tools@v0.0.1-2020.1.6/ir/lift.go:208 +0x525
honnef.co/go/tools/ir.(*Function).finishBody(0xc00b3e0000)
        honnef.co/go/tools@v0.0.1-2020.1.6/ir/func.go:565 +0x356
honnef.co/go/tools/ir.(*builder).buildFunction(0xc002155a48, 0xc00b3e0000)
        honnef.co/go/tools@v0.0.1-2020.1.6/ir/builder.go:2313 +0x485
honnef.co/go/tools/ir.(*builder).buildFuncDecl(0xc002155a48, 0xc006bb0120, 0xc014f48ba0)
        honnef.co/go/tools@v0.0.1-2020.1.6/ir/builder.go:2334 +0xff
honnef.co/go/tools/ir.(*Package).build(0xc006bb0120)
        honnef.co/go/tools@v0.0.1-2020.1.6/ir/builder.go:2440 +0xcc5
sync.(*Once).doSlow(0xc006bb0178, 0xc01a9b9bb0)
        sync/once.go:66 +0xec
sync.(*Once).Do(...)
        sync/once.go:57
honnef.co/go/tools/ir.(*Package).Build(0xc006bb0120)
        honnef.co/go/tools@v0.0.1-2020.1.6/ir/builder.go:2358 +0x65
honnef.co/go/tools/internal/passes/buildir.run(0xc016e535f0, 0x2e3f6805a, 0x1676100, 0xc00c0892c8, 0x2)
        honnef.co/go/tools@v0.0.1-2020.1.6/internal/passes/buildir/buildir.go:89 +0x298
github.com/golangci/golangci-lint/pkg/golinters/goanalysis.(*action).analyze(0xc0074e7130)
        github.com/golangci/golangci-lint/pkg/golinters/goanalysis/runner.go:590 +0xa83
github.com/golangci/golangci-lint/pkg/golinters/goanalysis.(*action).analyzeSafe.func2()
        github.com/golangci/golangci-lint/pkg/golinters/goanalysis/runner.go:512 +0x2a
github.com/golangci/golangci-lint/pkg/timeutils.(*Stopwatch).TrackStage(0xc001d6f8b0, 0xe8ccb4, 0x7, 0xc00499df70)
        github.com/golangci/golangci-lint/pkg/timeutils/stopwatch.go:111 +0x50
github.com/golangci/golangci-lint/pkg/golinters/goanalysis.(*action).analyzeSafe(0xc0074e7130)
        github.com/golangci/golangci-lint/pkg/golinters/goanalysis/runner.go:511 +0x91
github.com/golangci/golangci-lint/pkg/golinters/goanalysis.(*loadingPackage).analyze.func2(0xc011aaa720, 0xc0074e7130)
        github.com/golangci/golangci-lint/pkg/golinters/goanalysis/runner.go:1059 +0x65
created by github.com/golangci/golangci-lint/pkg/golinters/goanalysis.(*loadingPackage).analyze
        github.com/golangci/golangci-lint/pkg/golinters/goanalysis/runner.go:1054 +0x305 
WARN [runner] Can't run linter goanalysis_metalinter: buildir: package "mappercommon" (isInitialPkg: true, needAnalyzeSource: true): runtime error: invalid memory address or nil pointer dereference 
ERRO Running error: buildir: package "mappercommon" (isInitialPkg: true, needAnalyzeSource: true): runtime error: invalid memory address or nil pointer dereference 
```

</details>

**Comment by daixiang0** [comment]
Oh I see, if can not continue, the first one is better.

**Comment by ldez** [comment]
Closed in favor of dominikh/go-tools#882
